src/test/java/com/epam/atm/selenium/LoginActionTest.java

```diff
@@ -0,0 +1,23 @@
+package com.epam.atm.selenium;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pageobject.HomeActionPage;
+import pageobject.HomePage;
+import pageobject.LoginActionsPage;
+import pageobject.LoginPage;
+
+public class LoginActionTest extends BaseActionTest {
+    private static final String TITLE_HOME_PAGE = "Home | Salesforce";
```

Comment:
We need change a bit this verification:
Move private static final String TITLE_HOME_PAGE = "Home | Salesforce"; to your HomePage class

add method  in HomePage 
public boolean isPageTitleValid(){
return TITLE_HOME_PAGE .equals(getTitleOfHomePage())
}

in your test change
Assert.assertTrue(homePage.isPageTitleValid(), "User is not logged in or incorrect Home page is not opened");

Please correct same assertions in other tests